pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -109,6 +110,8 @@ func validateStart(ctx context.Context, t *testing.T, c *Cluster) {
 		assert.Equal(t, map[string]string{"my": "annotation"}, d.Spec.Template.Annotations)
 		assert.Contains(t, d.Spec.Template.Labels, "my-label-key")
 		assert.Equal(t, "my-priority-class", d.Spec.Template.Spec.PriorityClassName)
+		vars := k8sutil.FindDuplicateEnvVars(d.Spec.Template.Spec)
```

Comment:
Can this test be added to the generic unit tests that get run against Ceph deployments instead since it's broadly applicable?

`rook/pkg/operator/ceph/test/containers.go`

Reply:
And `makeSetServerAddrInitContainer` sets it an additional time as well.

Reply:
@BlaineEXE 

Thank you for your comment!
You are absolutely right, so I moved `FindDuplicateEnvVar` function to `rook/pkg/operator/ceph/test/containers.go`